src/components/VmDetails/cards/SnapshotsCard/index.js

```diff
@@ -75,7 +80,7 @@ const SnapshotsCard = ({ vm }) => {
       idPrefix={idPrefix}
       editable={false}
     >
-      <ConnectedSnapshots snapshots={snapshots} vmId={vm.get('id')} canUserManipulateSnapshot={vm.get('canUserManipulateSnapshots')} idPrefix={idPrefix} />
+      <ConnectedSnapshots snapshots={snapshots} vmId={vm.get('id')} canUserManipulateSnapshot={vm.get('canUserManipulateSnapshots')} idPrefix={idPrefix} isVmDown={vm.get('status') === DOWN_STATUS} />
```

Comment:
Nit-pick: This line is too long.  Splitting it to have each prop to its own line would be easier to read and have consistent styling with the rest of the codebase.